style.css

```diff
@@ -0,0 +1,157 @@
+@import url("./components/Button/button.css");
+@import url("./components/Avatar/avatar.css");
+@import url("./components/Alert/alert.css");
+@import url("./components/Badge/badge.css");
+@import url("./components/Lists/lists.css");
+@import url("./components/Card/card.css");
+@import url("./components/Image/image.css");
+@import url("./components/Input/input.css");
+@import url("./components/Navigation/navigation.css");
+@import url("./components/Rating/rating.css");
+@import url("./components/Modal/modal.css");
+:root {
+  --color-primary: #20242d;
+  --color-secondary: #78797a;
+  --color-accent: #60dafd;
+  --color-links: #bcbdbc;
+  --color-danger: red;
+  --color-difblue: #115e85;
+  --color-offwhite: #f3f3f3;
+  --color--lightblack: #333;
+}
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+/* Typography */
+html {
+  font-size: 62.5%;
+}
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 2rem;
+  background-color: var(--color-secondary);
+}
+h1,
+h2,
+h3 {
+  /* color: var(--color-accent); */
+  margin-bottom: 1rem;
+  line-height: 1.1;
+}
+h1 {
+  font-size: 5rem;
+}
+h2 {
+  font-size: 3rem;
+}
+h3 {
+  font-size: 1.5rem;
+}
+a {
+  text-decoration: none;
+  color: var(--color-links);
+}
+
+.navbar,
+.navbar__list {
+  display: flex;
+}
+.navbar {
+  justify-content: space-around;
+  flex-grow: 1;
+  background-color: var(--color-primary);
+  padding: 1.4rem;
+  min-height: 10vh;
+  align-items: center;
+}
+.navbar__list {
+  width: 30%;
+  justify-content: space-around;
+  list-style: none;
+  /* flex-grow: 1; */
+}
+.navbar__list li {
+  margin-right: 0.5rem;
+}
+.navbar__logo {
+  color: var(--color-accent);
+}
+.hero {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  min-height: 90vh;
+  background-color: var(--color-secondary);
+}
+.hero h1,
+p {
+  padding: 2rem;
+  text-align: center;
+}
+.btn__hero {
+  outline: none;
+  border: none;
+  padding: 1.5rem 2rem;
+  background-color: var(--color-accent);
+  color: var(--color-primary);
+  font-size: 2rem;
+  cursor: pointer;
+}
+h1.hero__title {
+  color: var(--color-accent);
+}
+.aside {
+  position: fixed;
+  background-color: var(--color-offwhite);
+  min-height: 90vh;
+}
+.aside__list li > a,
+.aside {
+  padding: 0.5rem;
+}
+.aside__list li {
+  list-style: none;
+  padding: 0.5rem;
+}
+.component-links > a {
+  color: var(--color--lightblack);
+}
+.component-links > a:hover,
+component-links > a:focus {
+  background-color: var(--color-accent);
+  color: var(--color-primary);
+}
+
+/* .grid--section {
+  padding: 2rem;
+  display: grid;
+  grid-template-columns: 0.5fr 1fr 0.5fr;
+  grid-template-areas: "aside main sidebar";
+   } */
+/* .aside {
+  grid-area: "aside";
+}
+.main {
+  grid-area: "main";
```

Comment:
you can remove those code which is commented and unnecessary!

Reply:
Thanks it is Done